dxp/how_clay_arrives_to_dxp.md

```diff
@@ -0,0 +1,223 @@
+# How Clay arrives to DXP?
+
+The goal of this section is to explain how Clay arrives to DXP, what's the correct way to enable it into your theme or module and what options do we have once imported.
+
+## How is Clay brought into DXP?
+
+There are 2 main methods to import Clay into DXP:
```

Comment:
We should clarify this more. Every module including themes use Clay CSS from `modules/node_modules`. Themes copy the files from there into `src/css/clay`. Clay CSS is included on the page through the file `clay.scss` from the theme. Any modification to variables, mixins, functions in the theme won't translate to DXP modules.

DXP modules reference those files directly mostly for Clay CSS variables, functions, and mixins.